dinky-cdc/dinky-cdc-core/src/main/java/org/dinky/cdc/AbstractSinkBuilder.java

```diff
@@ -131,7 +131,10 @@ protected Properties getProperties() {
                     && entry.getKey().startsWith("properties")
                     && Asserts.isNotNullString(entry.getValue())) {
                 properties.setProperty(entry.getKey().replace("properties.", ""), entry.getValue());
+            } else {
+                properties.setProperty(entry.getKey(), entry.getValue());
```

Comment:
because there's no place in the constructor of kafkaSinkBuilder to set properties.
![image](https://github.com/user-attachments/assets/8c36dcc9-3a40-45a0-9abb-3ae3f9b47edf)

Reply:
Thx. I get it.